languages/javalang/builder-renderer/commercetools-java-client/src/main/java/client/TempClient.java

```diff
@@ -0,0 +1,8 @@
+package client;
+
+public class TempClient implements HttpClient {
+    
+    public HttpResponse execute(final HttpRequest httpRequest) {
+        return new HttpResponse(200, null, "{}");
```

Comment:
What's the purpose of this client?

Reply:
Just to have any dummy client to be able to see how everything fits together. I will change its name and add functionality in the next PR. It will be like a "default" client that is used if no other is provided.